### PR TITLE
changed the contribute data ui

### PR DESCRIPTION
**Pull request by khavinshankar**
issue #150, New UI for the "Contribute Verified Data" page

### CODE REVIEW

File: pages/data.js [hunk 1]
```diff
@@ -78,57 +78,30 @@ export default function DataForm() {
 
     return (
         <div>
-            <section className="flex flex-col items-center md:pt-5">
```
Comment (Deep-Codes) [inline pages/data.js:81]: Add padding or anything so that header theme btn & lang selector don't overlap on Mobile Screens

**Review by Deep-Codes — CHANGES_REQUESTED** [review]
ss

On Safari Looks like this ....

On Safari Macos

Screenshot 2021-04-27 at 8 06 53 PM

**Comment by Deep-Codes** [comment]
@khavinshankar  Is autoprefixer working ?

File: pages/data.js (hunk 1 above)
Comment (Deep-Codes) [reply pages/data.js:81]: add `pt-10` instead of `md:pt-5`

**Comment by khavinshankar** [comment]
> @khavinshankar Is autoprefixer working ?

not sure

**Comment by Deep-Codes** [comment]
@khavinshankar still having the same problem

I would see something from my side. :(

**Comment by joelsajichacko** [comment]
On Desktop Chrome browser, it look like this
If possible ,please make this look similar to Safari Mac Os. Now the size of card is small and also the change the color of subtext..its too hard to read.

Screenshot (357)

**Pull request merged**